client/src/pages/Dashboard.js

```diff
@@ -47,29 +51,43 @@ const pastInterviews = [
 ];
 
 const Dashboard = () => {
+	const [open, setOpen] = useState(false)
+	const {upcomingInterviews, WaitingRoomOpen, setWaitingRoomOpen} = useContext(UserContext)
+
 	return (
-		<Container maxWidth="lg">
-			<Box pt={8}>
-				<Typography
-					style={{ color: theme.palette.primary.light }}
-					variant="h4"
-					align="center"
-				>
-					Upcoming practice interviews
-				</Typography>
-				<UpcomingInterviews rows={upcomingInterviews} />
-			</Box>
-			<Box pt={8} pb={12}>
-				<Typography
-					style={{ color: theme.palette.primary.light }}
-					variant="h4"
-					align="center"
-				>
-					Past practice interviews
-				</Typography>
-				<PastInterviews rows={pastInterviews} />
-			</Box>
-		</Container>
+		<>
+			<Container maxWidth="lg">
+				<button onClick={() => setOpen(true)}>Start</button>
```

Comment:
Can you update this to more closely match the styling for the rest of the app (using Material UI button)? Maybe changing the text to something like Create Interview as well